src/main/java/com/webtutsplus/ecommerce/controller/OrderController.java

```diff
@@ -55,4 +55,12 @@
         return new ResponseEntity<StripeResponse>(stripeResponse,HttpStatus.OK);
     }
 
+    @GetMapping("/{id}")
+    public ResponseEntity<Optional<Order>> getAllOrders(@PathVariable("id") Integer id, @RequestParam("token") String token) throws AuthenticationFailException {
```

Comment:
dont send optional, send order or throw exception that order does not exits

Reply:
Done